lib/cli/utils.js

```diff
@@ -88,7 +88,8 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   // they'll be not valid (those are completions for previous inputs)
   if (newDepth <= inputs.length - 1) completions = [];
   if (!completions[0] ||
-      completions.length === 1 && completions[0] === inputs[depth]) {
+    completions.length === 1 && completions[0] === inputs[depth]
```

Comment:
If I remember correctly we decided that this part will look like that:
```js
  if (!completions[0] ||
     (completions.length === 1 &&
      completions[0] === inputs[depth])) {
```
Correct me if I'm wrong.